### PR TITLE
chore: move Apollos to node 12

### PR DESCRIPTION
**Pull request by redreceipt**

### CODE REVIEW

**Review by vinnyjth — APPROVED** [review]
I dig it. Merge whenever you get the tests passing 👍

**Comment by redreceipt** [comment]
yea they're passing locally. this is frustrating.

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "codecov": "3.7.1",
     "conventional-changelog-conventionalcommits": "^4.5.0",
     "eslint-plugin-prettier": "^3.1.4",
-    "lerna": "^3.14.0",
+    "lerna": "^4.0.0",
```
Comment (redreceipt) [inline package.json:46]: whoa

Comment (redreceipt) [reply package.json:46]: this PR is no joke now

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -39,7 +39,7 @@ jobs:
       run: yarn
 
     - name: Jest
-      run: yarn test --stream -- -- --maxWorkers=2
+      run: yarn test --stream --concurrency=1 --max-buffer=10000000 -- -- --maxWorkers=2
```
Comment (redreceipt) [inline .github/workflows/ci.yml]: maybe we could just get away with `yarn test` to match local runs?

Comment (vinnyjth) [reply .github/workflows/ci.yml]: That's my eventual goal. I want to get to the point where I can see the test failures (they aren't truncated) and then work backwards from there.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -39,19 +33,13 @@ jobs:
...
-    - name: Install Yarn
-      run: npm install -g yarn
+    - uses: actions/setup-node@v2
 
     - name: Install Dependencies
       run: yarn
 
     - name: Jest
-      run: yarn test --stream -- -- --maxWorkers=2
+      run: yarn test --no-bail --stream --concurrency=1 --max-buffer=10000000 -- -- --maxWorkers=2
```
Comment (vinnyjth) [inline .github/workflows/ci.yml:42]:
```suggestion
      run: yarn test --stream -- -- --maxWorkers=2
```

Comment (vinnyjth) [inline .github/workflows/ci.yml:42]:
```suggestion
      run: yarn test --no-bail --concurrency=1 --max-buffer=10000000 -- -- --maxWorkers=2
```

**Comment by vinnyjth** [comment]
Now that you can see failures, want to take another stab at tests?

**Pull request merged**